static/app/views/integrationPipeline/init.tsx

```diff
@@ -0,0 +1,7 @@
+import 'focus-visible';
```

Comment:
Is this used elsewhere, and included here for bundling reasons?

Reply:
This is existing code that was moved https://github.com/getsentry/sentry/pull/26546/files#diff-5dd55674f923db81c8704a9b1f4598a45bdc5f5012f4c5e6127aa0544eb87406L1

Basically just added a level of indirection by moving existing `index` files --> `init`, and having `index` import init